xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/ledger/LedgerRequestParams.java

```diff
@@ -78,9 +78,9 @@ default boolean transactions() {
    * Provide full JSON-formatted information for transaction/account information instead of only hashes.
    * Defaults to false. Ignored unless you request {@link #transactions()}, {@link #accounts()}, or both.
    */
-  @Value.Default
+  @Value.Derived
```

Comment:
Minor nit but if these are derived they should probably be marked `@Overide`

Reply:
They aren't overriding anything though?

Reply:
Oh, I'm confused then - these "Derived" values will only ever have a single value, but the function makes it look like these might have two values (true or false). The Javadoc also makes this seems like this field is a shared concept.